examples/mobile/build.rs

```diff
@@ -0,0 +1,27 @@
+//! A build script to copy libc++_shared.so to the target directory.
+use std::{
+    env,
+    path::{Path, PathBuf},
+};
+
+fn main() {
+    if env::var("CARGO_CFG_TARGET_OS").unwrap() == "android" {
+        android();
+    }
+}
+
+fn android() {
+    println!("cargo:rustc-link-lib=c++_shared");
```

Comment:
this shouldn't be needed, Bevy itself doesn't link to libc++, oboe does, and oboe already sets the proper link commands for it

Reply:
There is a workaround, but it is rather complex

You need a empty cpp file, setup cmake, then tell gradlew to link it in.

and for now I can't control which target it will link, so it will simply link all `libc++_shared.so` in all targets, even if you didn't meant to add it in, with an extra empty file(like `libdummy.so`).